### ISSUE TITLE
 Is it possible to open new tabs without the context menu without being frozen?

### ISSUE THREAD

**Issue by AlexMak58** [deanoemcke/thegreatsuspender#655]
Hello.
Is it possible to open new tabs without the context menu without being frozen?
It is very inconvenient to open each tab with the context menu. I want to make it easier. It's real?

**Comment by deanoemcke** [comment]
Sorry, I dont understand your question.
Why are you opening new tabs via the context menu?
It doesn't sound like your issue has anything to do with this extension?

**Comment by AlexMak58** [comment]
I want to be able to open the tabs frozen without the context menu. Is this possible?

**Comment by deanoemcke** [comment]
This is possible - depending on the implementation. I don't think I can specifically override the 'ctrl+click' functionality, but i could add an option so that all new tabs that are opened but not given focus are automatically suspended. Is that what you are looking for?
I'm just not sure this would be a good idea as it could result in some confusion for users..

**Comment by AlexMak58** [comment]
Yes, it would be nice to see this implementation. I would be pleased.

**Comment by dumblob** [comment]
Raising my hand for this (currently I live with `Native Lazy Tabs` extension for this).

**Comment by deanoemcke** [comment]
How about just setting the 'time to suspend' to 20 seconds? Does that give you a similar outcome?

**Comment by dumblob** [comment]
@deanoemcke unfortunately not, because in my case the main purpose is to prevent loading of hundreds of tabs after a Chromium (re)start. Without that it gets extremely slow, clunky, downloads hundreds of megabytes of data immediately (even though I won't use half of the tabs in the next few hours) and is way more prone to session loss in case something bad happens to the browser process(es) (it's not a theory - I've lost my "tabs" already several times before I started using `Native Lazy Tabs`).

**Comment by deanoemcke** [comment]
@dumblob it sounds like you're speaking of a slightly different feature. that is, automatically suspend all tabs on startup. this is already a separate issue: https://github.com/deanoemcke/thegreatsuspender/issues/49

I'll add some info about Native Lazy Tabs there.

**Comment by dumblob** [comment]
@deanoemcke well, it's though the most frequent use case of mine, but curiously this issue caught my eye, because I'm very often using the workflow in which I open e.g. 60 new tabs from a search engine page and first then I'm checking what is in those tabs. And because opening the links one after another very quickly has the same effect as when I (re)start Chromium, I mentioned the restart as it's easier to comprehend.

**Comment by dumblob** [comment]
>for this new scenario, are you aware of the right-click 'open in new suspended tab' option?

Well, I'm using TGD (which doesn't support this option) for about two years already and forgot about this option in TGS. Thank you for the reminder.

That would definitely cover part of the cases. All the rest would be complicated as I'm using cVim and therefore it would be useful for me to either allow cVim to run the function from TGS (which is IMHO not allowed due to security reasons) or just making this functionality independent from any extension (e.g. by autosuspending/autodiscarding the tab immediately after it has been opened in the background which is what this GitHub issue is basically about).
